src/hooks/multiSig/useEligibleSignees.ts

```diff
@@ -32,18 +32,21 @@ export const useEligibleSignees = ({
     signeesPerRole: {},
   });
 
-  // Since it's an array fetched from a function, we "memo" this by stringify it to prevent it always triggering the useEffect
+  // Since they are both arrays fetched from a function, we "memo" this by stringifying them to prevent always triggering the useEffect
   const requiredRolesString = JSON.stringify(requiredRoles);
+  const domainIdsString = JSON.stringify(domainIds);
 
   useEffect(() => {
     async function fetchEligibleSignees() {
       const parsedRequiredRoles = JSON.parse(requiredRolesString);
+      const parsedDomainIds = JSON.parse(domainIdsString);
+
```

Comment:
Given this `useEffect` gets triggered each time `requiredRolesString` or `domainIdsString` changes, we should have access to the latest values from which we created these strings (`requiredRoles` and `domainIds`) so we could directly use those here, without the need to parse the stringified value